src/main/java/pl/exsio/querydsl/entityql/Q.java

```diff
@@ -194,4 +204,12 @@ void addPrimaryKey(List<QEntityColumnMetadata> ids) {
     public PrimaryKey<E> primaryKey() {
         return (PrimaryKey<E>) id;
     }
-}
\ No newline at end of file
+
+    private QEntityScanner qEntityScannerCheck() {
+        if (!(scanner instanceof QEntityScanner)) {
+            throw new UnsupportedScannerException(scanner);
```

Comment:
please add String description with the type of operation that won't be supported

Reply:
Added description